packages/core/src/Cardano/types/Utxo.ts

```diff
@@ -15,7 +16,30 @@ export interface HydratedTxIn extends TxIn {
 export interface TxOut {
   address: Address;
   value: Value;
-  datum?: Hash32ByteBase16;
+
+  /**
+   * Datum hash, this allows to specify a Datum without publicly revealing its value. To spend an output which specifies
+   * this type of datum, the actual Datum value must be provided and will be added to the witness set of
+   * the transaction.
+   */
+  datumHash?: Hash32ByteBase16;
+
+  /**
+   * Inline datum reveals the value on the blockchain at the time of output creation. This way of attaching datums lets
+   * users consume this output without specifying the datum value.
```

Comment:
They are still specifying the datum value, but they can find it on-chain.
```suggestion
   * users consume this output without specifying the datum value separately.
```

Reply:
Right, you only specify `{txId,index}`